recipes/holopy/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ $(uname) == Darwin ]; then
+    export CFLAGS="-arch x86_64"
+    export FFLAGS="-static -ff2c -arch x86_64"
+    export LDFLAGS="-Wall -undefined dynamic_lookup -bundle -arch x86_64"
+    export LDFLAGS="-Wl,-search_paths_first -L$(pwd) $LDFLAGS"
```

Comment:
I copied this file from another feedstock compiling Fortran code (I can't remember which one). I believe it has something to do with specifying where to find gfortran libraries. 

The code doesn't compile properly if I don't include it.